### ISSUE TITLE
Syntax Request

### ISSUE THREAD

**Issue by mobergmk** [elvirbrk/NoteHighlight2016#11]
Installed but haven't used it yet, do you plan to add PowerShell at some point? Awesome add-on, I look forward to trying it out next week.

**Comment by elvirbrk** [comment]
Yes, there is plan to include support for all languages supported by "highlight" tool, including PowerShell.

**Comment by elvirbrk** [comment]
Version v3.0 has support for PowerShell, just see readme for how to enable it by default.

**Comment by khuongduybui** [comment]
I added a few 48x48 png files under Resources but the app can't seem to reference them. Probably need to edit some dll too?

**Comment by elvirbrk** [comment]
> I added a few 48x48 png files under Resources but the app can't seem to reference them. Probably need to edit some dll too?

Yes, icons need to be included into application Resource dll. If that's something you created and would like to share, than I can include them.
But I will also see into adding option to include custom icons.

**Comment by Davito3** [comment]
Not sure if this should be moved into another feature request, but I was wondering if there's any updates on the custom icon feature (since idk how to change dll's)

**Comment by elvirbrk** [comment]
Not yet, but if you have some custom icons, attach them here and I'll include them in the project.
